things/thing-7.md

```diff
@@ -0,0 +1,158 @@
+    citation metrics are important to the academic community. Find out where
+    data fits in the citation picture.
+keywords: |
+    Thing 7, data citation, citation analysis, citation metrics,data,
+    research, research data, australia
+og:description: |
+    Thing 7: Data citation for access & attribution. Citation analysis and
+    citation metrics are important to the academic community. Find out where
+    data fits in the citation picture.
+og:site_name: ANDS
+og:title: 'Thing 7: Data citation for access &amp; attribution'
+og:type: website
+og:url: 'https://www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-7'
+robots: 'index,follow'
+title: 'Thing 7: Data citation for access & attribution - ANDS'
+viewport: |
+    width=device-width, initial-scale=1.0, maximum-scale=1.0,
+    user-scalable=0
+---
+#Thing 7: Data citation for access & attribution
+
+**[&lt;&lt;Jump to Thing 6](thing-6.md) &nbsp; &nbsp; [Jump to Thing 8&gt;&gt;](thing-8.md)**
+
+Citation analysis and citation metrics are important to the academic
+community. Find out where data fits in the citation picture.
+
+-   **Getting started**: how to cite data
+-   **Learn more**: may the FORCE(11) be with you (while you cite data)
+-   **Challenge me**: are some data formats more likely to be cited?
+
+
+* [Getting started](#getting-started)
+* [Learn more](#learn-more)
+* [Challenge me](#challenge-me)
+
+## Getting started 
+
+### Citing research data
+
+Data citation continues the tradition of acknowledging other people’s
+work and ideas. Along with books, journals and other scholarly works, it
+is now possible to formally cite research datasets and even the software
+that was used to create or analyse the data.
+
+1.  Start by looking back to the [Weddell Seal
+    dataset](https://researchdata.ands.org.au/measuring-effects-human-leptonychotes-weddellii/640511 "Wendell seal dataset")
+    we explored in Thing 4.  Check out how many times it has been cited.
+    This citation count has been measured by Thomson Reuters Data
+    Citation Index product.
+2.  Scan through the ANDS introduction to [data
+    citation](https://www.ands.org.au/working-with-data/citation-and-identifiers/data-citation "data citation")
+3.  Now look at the [Hutchinson Drought
+    Index](https://researchdata.ands.org.au/monthly-drought-australia-drought-index/61872 "Hutchinson drought index")
+    data record in Research Data Australia. This research data makes
+    cross disciplinary connections between episodes of drought and
+    correlated increases in rural mental health issues. The beauty of
+    this record is that it shows the entirety of the research outputs -
+    publications, software, related datasets and more - all of which are
+    citable.
+4.  Click on the ‘Cite’ button to see the similarities between the
+    formats for citation of data and other scholarly publications.  Did
+    you notice that, as yet, there are no citation metrics to this
+    record?
```

Comment:
_"Did you notice that, as yet, there are no citation metrics to this  record?_  
This text may go out of date at any time - e.g. no citations today but X citations next week.  Also, the action undertaken by the user at this point (open citation panel) will not reveal citation count.  Suggest reworking this sentence as a question (e.g. Has this dataset been cited?) and shifting the reworked sentence to the previous bullet point

Reply:
Fixed as suggested